rpgJavaInterpreter-core/src/test/resources/db/WRITE02.rpgle

```diff
@@ -0,0 +1,23 @@
+     FEMPLVIEW  IF   E           K DISK
```

Comment:
The file type is incorret: I = input.
The corret way to declare this file is:
FEMPLVIEW  UF A E           K DISK
because U = update and A is used to specified the possibility of adding new record to  file.

Reply:
Take in account that jariko ignores all those keywords, however I will fix.

Reply:
Ok

Reply:
Try to verify because I am not sure of what i have done, if you need some suggestion about how to review my changes I am at your disposition